docker/Makefile

```diff
@@ -3,18 +3,48 @@ ROOT_IMAGE ?= alpine:3.12
 CERT_IMAGE := alpine:3.12
 GOLANG_IMAGE := golang:1.15-alpine
 
-BASE_IMAGE := localhost/baseimg:$(VERSION)-$(shell echo $(ROOT_IMAGE) | tr : -)
-DEBUG_IMAGE := localhost/debugimg:$(VERSION)-$(shell echo $(GOLANG_IMAGE) | tr : -)
+BASE_IMAGE := $(DOCKER_NAMESPACE)/baseimg:$(VERSION)-$(shell echo $(ROOT_IMAGE) | tr : -)
+DEBUG_IMAGE := $(DOCKER_NAMESPACE)/debugimg:$(VERSION)-$(shell echo $(GOLANG_IMAGE) | tr : -)
 
 create-baseimg-debugimg: create-baseimg create-debugimg
 
+dockerx-baseimg-debugimg: .ensure-docker-buildx .ensure-buildx-builder
+	BUILDX_PUSH=true $(MAKE) create-baseimg-debugimg
+
 create-baseimg:
-	docker build -t $(BASE_IMAGE) \
+	$(DOCKER_BUILD) -t $(BASE_IMAGE) \
 		--build-arg root_image=$(ROOT_IMAGE) \
 		--build-arg cert_image=$(CERT_IMAGE) \
 		docker/base
 
 create-debugimg:
-	docker build -t $(DEBUG_IMAGE) \
+	$(DOCKER_BUILD) -t $(DEBUG_IMAGE) \
 		--build-arg golang_image=$(GOLANG_IMAGE) \
 		docker/debug
+
+# Ensure "docker buildx" is available and enabled. For more details, see: https://github.com/docker/buildx/blob/master/README.md
+.ensure-docker-buildx:
```

Comment:
Why the dot in the name? Also, a beter name would probably be `install-docker-buildx`. To me, `ensure` would convey that it would fail in case the plugin isn't installed.

Reply:
ensure means install if need.